infra/cifuzz/continuous_integration.py

```diff
@@ -191,6 +196,23 @@ def build_external_project_docker_image(project_name, project_src,
   return helper.docker_build(command)
 
 
+class ExternalGeneric(BaseCi):
+  def get_diff_base(self):
+    return 'origin...'
+
+  def prepare_for_fuzzer_build(self):
+    logging.info('YAAAAAAAY')
+    manager = repo_manager.RepoManager(self.config.project_src_path)
+    abs_build_integration_path = os.path.join(manager.repo_dir,
+                                              self.config.build_integration_path)
+    if not build_external_project_docker_image(
+        self.config.project_name, manager.repo_dir, abs_build_integration_path):
+      logging.error('Failed to build external project.')
+      return BuildPreparationResult(False, None, None)
```

Comment:
nit:named args is better for readability.
xyz=False, abc=None, def=None

Reply:
Done.